### PR TITLE
Test builtins

### PR DESCRIPTION
**Pull request by karol-bisztyga**
Testing cairo code that uses builtins.

### CODE REVIEW

**Review by Arcticae — COMMENTED** [review]
It's only testing if the binding works, can we try running this code on our vm? 

Also, if it's not supported in tests themselves (and i suppose that it should, from the comment above), can we link an issue?

File: protostar/cairo/cairo_function_runner_facade.py [hunk 1]
```diff
@@ -42,6 +49,61 @@ def run_from_offset(
...
+        apply_modulo_to_args - Apply modulo operation on integer arguments.
+        """
+        if hint_locals is None:
+            hint_locals = {}
+
+        if verify_secure is None:
+            verify_secure = True
+
+        if apply_modulo_to_args is None:
+            apply_modulo_to_args = True
```
Comment (karol-bisztyga) [inline protostar/cairo/cairo_function_runner_facade.py:81]: This logic can be moved to default arguments' values.

File: tests/integration/cairo1/builtins_test.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    assert protostar_casm_json
+    protostar_casm = ProtostarCasm.from_json(protostar_casm_json)
+    cairo_runner_facade = CairoRunnerFacade(program=protostar_casm.program)
+
+    for name, offset in protostar_casm.offset_map.items():
+        cairo_runner_facade.run_from_offset(offset=offset)
+        if name.endswith("_panic"):
+            assert cairo_runner_facade.did_panic()
+        else:
+            assert not cairo_runner_facade.did_panic()
```
Comment (karol-bisztyga) [inline tests/integration/cairo1/builtins_test.py:26]:
```suggestion
        assert name.endswith("_panic") == cairo_runner_facade.did_panic()
```

File: tests/integration/cairo1/builtins_test/test.cairo [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    pedersen(1, 2);
+    assert(1 == 1, 'simple check');
+}
+
+
+#[test]
+fn test_pedersen_panic() {
+    pedersen(1, 2);
+    assert(1 == 2, 'simple check');
+}
```
Comment (karol-bisztyga) [inline tests/integration/cairo1/builtins_test/test.cairo:33]:
```suggestion
fn use_bitwise() {
    bitwise(1_u128, 1_u128);
}

fn use_pedersen() -> felt {
    assert(1 == 1, 'simple check');
    pedersen(1, 2)
}

#[test]
fn test_indirect_use() {
    use_pedersen();
    use_bitwise();
    assert(1 == 1, 'simple check');
}

#[test]
fn test_direct_use() {
    pedersen(1, 2);
    bitwise(1_u128, 1_u128);
    assert(1 == 1, 'simple check');
}


#[test]
fn test_pedersen_panic() {
    pedersen(1, 2);
    assert(1 == 2, 'simple check');
}
```

**Review by karol-bisztyga — COMMENTED** [review]
Approve, some nits pointed out

File: protostar/cairo/cairo_function_runner_facade.py (hunk 1 above)
Comment (MaksymilianDemitraszek) [reply protostar/cairo/cairo_function_runner_facade.py:81]: Idk why it is like this in run_from_entrypoint 🤔

Comment (MaksymilianDemitraszek) [reply protostar/cairo/cairo_function_runner_facade.py:81]: I will leave it as it is for consistency though

Comment (karol-bisztyga) [reply protostar/cairo/cairo_function_runner_facade.py:81]: Ok, sure

**Pull request merged**